server.js

```diff
@@ -6287,6 +6287,92 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// MicroBadger integration.
+camp.route(/^\/microbadger\/(image-size|layers)\/([^/]+)\/([^/]+)\/?([^/]*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var type = match[1];
+  var user = match[2];
+  var repo = match[3];
+  var tag = match[4];
+  var format = match[5];
+  if (user === '_') {
+    user = 'library';
+  }
+  var path = user + '/' + repo;
```

Comment:
Would this be a good candidate for using template strings?

Reply:
Yep, why not